types/text-diff-types.tsx

```diff
@@ -0,0 +1,16 @@
+import { Change } from 'diff';
+
+export interface DiffOptions {
```

Comment:
Since this is only really used in one file, lets move it to the file it is used in please